include/zephyr/drivers/input.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_INPUT_H_
+#define ZEPHYR_INCLUDE_DRIVERS_INPUT_H_
+
+/**
+ * @brief Input Driver Interface
+ * @defgroup input_driver_interface Input Driver Interface
+ * @ingroup io_interfaces
+ * @{
+ */
+
+#include <zephyr/kernel.h>
```

Comment:
not needed

Reply:
dropped, added `zephyr/toolchain/common.h`, will ditch the whole thing if you confirm we should get rid of the api stub, see the other comment